.gitignore

```diff
@@ -15,3 +15,4 @@ tags
 *.rdb
 redis-git
 appendonlydir/
+testmodule.so
```

Comment:
Glob to future proof with *.so - just in case? Or, not worth it?

Reply:
Updated.